meta-ov/recipes-core/psplash/files/psplash-start.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Start psplash boot splash screen
+DefaultDependencies=no
+RequiresMountsFor=/run
+
+[Service]
+Type=notify
+ExecStart=/usr/bin/psplash --angle "${ROTATION}"
+EnvironmentFile=/etc/rotation
```

Comment:
Another solution that comes to mind is to run some shell script instead of /usr/bin/psplash directly, where we can find the correct rotation and pass it as `--angle` parameter.

Reply:
I wasted hours on this... The problem is that psplash-start communicates with psplash-systemd via a socket. (Hence type notify) it also expects it to remain after exit.
A shell wrapper destroys that functionality and even leaves ov unbootable. 

The only way to pass an env variable inside a systemd unit is via Environment or as above EnvironmentFile. Environmentfile has to exist before any process is started. So you cannot have a ExecStartPre process that creates a variable file. 

So the only path left is to write another unit that runs before psplash and wites the environment file. (i'm doing that now.)

Reply:
> I wasted hours on this... The problem is that psplash-start communicates with psplash-systemd via a socket. (Hence type notify) it also expects it to remain after exit. A shell wrapper destroys that functionality and even leaves ov unbootable.

Not if you `exec` the final program.